nose_setenv/nose_setenv.py

```diff
@@ -34,7 +41,7 @@ def configure(self, options, conf):
             return
 
         env_variables_to_override = ast.literal_eval(options.set_env_variables)
-        for env_key, env_val in env_variables_to_override.iteritems():
+        for env_key, env_val in _iteritems(env_variables_to_override):
```

Comment:
@mahmoudimus I addressed your comments here, by using the iteritems method and in case is not available (on python3) will use items.